packages/auto-approve/src/check-pr.ts

```diff
@@ -65,12 +66,16 @@ export async function checkPRAgainstConfig(
     // changedFiles and maxFiles are not set in the JSON schema
     let filePathsMatch = true;
     let fileCountMatch = true;
-    const versionChecks = true;
+    let addtlRules = true;
+    const releasePRFiles: File[] = [];
 
     // Since there's only one allowed title per author right now, we don't need to
     // add complicated logic to see which title should match the incoming PR; but,
     // this could be logic we work into the future.
     if (!title.match(rulesToValidateAgainst.title)) {
+      logger.info(
+        `Info for ${repoOwner}/${repo}/${prNumber} title does not match what is allowed`
```

Comment:
BTW, you fetch repoOwner from `pr.pull_request.head.repo.owner.login`, but it seems wrong.

For example, if I create a PR from my personal fork, the repoOwner will be `tmatsuo`.

Reply:
I guess this is why I only get auto-approve config check failure for PRs from the same repo and I don't get the failing check on PRs from my personal repo.

Reply:
Thanks, I corrected that!